docs/restrictions/PotentialsRE.md

```diff
@@ -0,0 +1,65 @@
+#Potentials for Renewable Energies
+
+##Research of renewable potentials in study comparison
+
+Based on a literature review, studies on renewable energy potentials were examined. Included are studies with the models GLAES/RESKit (cf. Franzmann et al. [1]) and PyGRETA (cf. Buchenberg et al. [2]), which have investigated the global renewable potentials within the project "ETSAP Deutschland" (cf. Figure 1 & Figure 2). 
+The comparison of the wind onshore studies with the results from the GLAES/RESkit and PyGRETA models is shown in Figure 1 can be seen. For wind onshore, some studies could be identified that investigate technical potentials based on GIS models with the addition of land use criteria. Studies that investigate the global technical potential based on all TIAM regions could not be found, yet there are some studies that evaluate several TIAM regions using the chosen approach. Among them are the studies by Bosch et al., Eurek et al., and Ruiz et al. [3–6] However, the studies here show significant differences in terms of results. For example, in these studies for Africa, the values, GLAES/RESKit and PyGRETA included, range from 601.5 GW to 149418.6 GW. This is partly because the TIAM Africa region aggregates the individual African countries. Many of the studies mentioned, however, only represent sub-areas of the TIAM regions, which makes a comparison difficult. If individual countries are compared, the picture is usually very similar; for example, the values for Germany in PyGRETA, GLAES/RESKit and Ruiz et al. [5] are close together at 283.8 GW, 294.77 GW and 308 GW (max. scenario). 
+In general, it can be said for the literature comparison based on onshore wind that the values are very scattered. The literature comparison cannot identify exactly to what extent the regional consideration has an influence on the results, but it has also been shown that on a country comparison the results of the literature almost match those of PyGRETA and GLAES/RESKit (cf. Mexico in Figure 1).
```

Comment:
```suggestion
In general, it can be said for the literature comparison based on onshore wind that the values are very scattered. The literature comparison cannot identify exactly to what extent the regional consideration has an influence on the results, but it has also been shown that on a country comparison the results of the literature almost match those of PyGRETA and GLAES/RESKit (cf. Mexico in Figure 1).

```